packages/client/webapp/src/stores/auth.ts

```diff
@@ -52,6 +52,7 @@ const useAuthStore = defineStore('auth', {
 					: await fetch('/silent-check-sso.html');
 
 			if (!response.ok) {
+				window.location.assign('http://localhost:8078/app/');
```

Comment:
my only concern here is the presence of `localhost`, does a relative path not work? i.e `/app`

Reply:
The relative path doesn't work because the localhost port for the webapp defaults to 8080 but keycloak is 8078.

Reply:
I think this need to be an `.env` variable to know what is the origin of the webapp.
Because on _staging_ and _production_ those will `terarium.ai/` and _local_ might stayt `localhost:8078/app`.

Reply:
Have to agree, hardcoding something like `localhost` is not great, and can be a source of errors to come.  Ideally, this should be relative, but I get that Yarn insists on a very specific developer environment, a shame as deployment can/will be much freer

Reply:
`/logout`, whilst effectively a no-op, is probably a nice "compromise" since the by product of logging out is being redirected to the front door.  As for the `http://localhost:8078` thing, it should be in the README for devs to find, and if someone doesn't like that specific port, then that can be changed I chose a non 8080 port with no rhyme or reason.

Reply:
How logout works and those "gateway" details are defined within the kubernetes yaml files and are editable.